mmedit/models/restorers/tdan.py

```diff
@@ -0,0 +1,143 @@
+import numbers
+import os.path as osp
+
+import mmcv
+
+from mmedit.core import tensor2img
+from ..builder import build_loss
+from ..registry import MODELS
+from .basic_restorer import BasicRestorer
+
+
+@MODELS.register_module()
+class TDAN(BasicRestorer):
+    """TDAN model for video super-resolution.
+
+    Paper:
+        TDAN: Temporally-Deformable Alignment Network for Video Super-
+        Resolution, CVPR, 2020
+
+    Args:
+        generator (dict): Config for the generator structure.
+        pixel_loss (dict): Config for pixel-wise loss.
+        train_cfg (dict): Config for training. Default: None.
+        test_cfg (dict): Config for testing. Default: None.
+        pretrained (str): Path for pretrained model. Default: None.
+    """
+
+    def __init__(self,
+                 generator,
+                 pixel_loss,
+                 lr_pixel_loss,
+                 train_cfg=None,
+                 test_cfg=None,
+                 pretrained=None):
+        super().__init__(generator, pixel_loss, train_cfg, test_cfg,
+                         pretrained)
+
+        self.lr_pixel_loss = build_loss(lr_pixel_loss)
+
+    def forward_train(self, lq, gt):
+        """Training forward function.
+
+        Args:
+            lq (Tensor): LQ Tensor with shape (n, c, h, w).
+            gt (Tensor): GT Tensor with shape (n, c, h, w).
+
+        Returns:
+            Tensor: Output tensor.
+        """
+        losses = dict()
+        output, aligned_lrs = self.generator(lq)
+
+        # loss on the HR image
+        loss_pix = self.pixel_loss(output, gt)
+        losses['loss_pix'] = loss_pix
+        # loss on the aligned LR images
+        t = aligned_lrs.size(1)
+        lr_ref = lq[:, t // 2, :, :, :].unsqueeze(1).repeat(1, t, 1, 1, 1)
```

Comment:
How about refactor this line to 
`lr_ref = lq[:, t // 2].unsqueeze(1).repeat(1, t, 1, 1, 1)`